src/Controller/UI/ReportDatasetSubmission.php

```diff
@@ -89,7 +89,7 @@ public function makeCSVAction(string $udi, DatasetRepository $datasetRepository)
             $label = $childView->vars['label'];
 
             if (empty($label)) {
-                continue;
+                $label = $childView->vars['name'] . ':';
```

Comment:
This is the only field that has a colon after it. Also, we should probably quote the label in the generated CSV.

Reply:
Removed, also removed from others, not needed because it's not consistent.